theme/apps/gdcmvs/dist/bundle.js

```diff
@@ -0,0 +1,2048 @@
+
+                    count_s = matched_pv.length;
+                } else {
+                    //matched on property name or description
+                    p.node = "";
+                    trs.push(p);
+                }
+            }
+
+            //save and calculate the count of matched element in this property
+            p.len = count_v + count_s;
+            c.len += p.len + count_p;
+            n.len += p.len + count_p;
+            result.len += p.len + count_p;
+        });
+
+        var offset = $('#root').offset().top;
+        var h = window.innerHeight - offset - 305;
+        h = h < 430 ? 430 : h;
+        var html = $.templates(__WEBPACK_IMPORTED_MODULE_0__view__["a" /* default */]).render({ mh: h, trs: trs });
+
+        result.html = html;
+        return result;
+    }
+};
+
+/* harmony default export */ __webpack_exports__["a"] = (func);
+
+/***/ }),
+/* 7 */
+/***/ (function(module, __webpack_exports__, __webpack_require__) {
+
+"use strict";
+
+var tmpl = '<div class="container table__container">' + '<div class="table__thead row">' + '<div class="col-xs-4"><div class="table__th">Name</div></div>' + '<div class="col-xs-4"><div class="table__th">Description</div></div>' + '<div class="col-xs-4"><div class="table__th table__th--right">' + '<div class="checkbox checkbox-th"><label class="checkbox__label">' + '<input class="checkbox__input" id="trs-checkbox" type="checkbox" value="">' + '<span class="checkbox__btn"><i class="checkbox__icon fa fa-check"></i></span> Show all values' + '</label></div>' + '<button type="button" id="tree_toggle" class="btn btn-th" data-toggle="button" aria-pressed="false" autocomplete="off">' + '<i class="btn-th__icon fa fa-angle-down"></i> Expand All' + '</button>' + '</div></div>' + '</div>' + '<div class="table__body table__body--overflow row" style="max-height: {{:mh}}px;">' + '<table class="treetable table" id="tree_table">' + '<tbody>' + '{{for trs}}' + '<tr key="{{:id}}" data-tt-id="{{:data_tt_id}}" data-tt-parent-id="{{:data_tt_parent_id}}" class="data-table-row {{:node}} {{if exist != true && type == "value"}}data-hide hide{{/if}}">' + '<td width="33%">' + '<span class="{{:type}} title table__td--word-break" style="display:inline-block; width: 70%;">' + '{{if type == "folder"}}' + '<a href="https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id={{:l_id}}" target="_blank">{{:title}} {{if len !== undefined}}({{:len}}){{/if}}</a>' + '{{else type == "link"}}' + '{{if l_type == "cde"}}' + 'No values in GDC, reference values in <a href="javascript:getCDEData(\'{{:l_id}}\');" class="table-td-link">caDSR</a>' + '{{else}}' + 'No values in GDC, concept referenced in <a target="_blank" href="{{:url}}" class="table-td-link">NCIt</a>' + '{{/if}}' + '{{else}}' + '{{:title}} {{if len && len !== 0}}({{:len}}){{/if}}' + '{{/if}}' + '</td>' + '<td width="66%">' + '{{:desc}}' + '</td>' + '</tr>' + '{{/for}}' + '</tbody>' + '</table>' + '</div>' + '</div>';
```

Comment:
don't think the `target=_blank` to the dictionary viewer is needed, since MVS is integrated into gdc-docs, it's strange to open a new tab for same site

Reply:
also is it possible to make this one really long line into a template file like https://github.com/NCIP/gdc-docs/blob/develop/theme/apps/dictionary/html-shards/views/entity-definition-controls.view.html?

Reply:
I didn't create the dictionary app, but you are right, is really long.
We need the target=_blank now, I know that is the same page but is a different application, we going to lose the last search in GDCMVS app.

Reply:
could you save/reload the last search from localStorage?

Reply:
@cy , I updated my branch, please could you help me to see I need other change. I removed `target=_blank` and I added localStorage.